### PR TITLE
fix: update to latest @octokit/* dependencies

### PR DESCRIPTION
**Pull request by gr2m**
closes https://github.com/googleapis/repo-automation-bots/pull/1195

- [x] buildcop
- [x] conventional-commit-lint
- [x] failurechecker
- [x] header-checker-lint
- [x] label-sync

### CODE REVIEW

**Comment by gr2m** [comment]
@googlebot I signed it!

File: packages/failurechecker/package.json [hunk 1]
```diff
@@ -28,11 +28,10 @@
     "lint": "gts check"
   },
   "dependencies": {
-    "@octokit/rest": "^18.0.6",
     "gcf-utils": "^6.2.0"
   },
   "devDependencies": {
-    "@octokit/types": "^5.5.0",
+    "@octokit/types": "^6.0.0",
```
Comment (JustinBeckwith) [inline packages/failurechecker/package.json:34]: Elsewhere this was replaced with `@octokit/openapi-types` - I'm curious - why keep this package here?  Is it still needed?

File: packages/monitoring-system/data-processor/package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     "@google-cloud/pubsub": "^2.6.0",
     "@google-cloud/tasks": "^2.1.2",
     "@octokit/rest": "^18.0.9",
-    "@octokit/types": "^5.5.0",
+    "@octokit/types": "^6.0.0",
```
Comment (JustinBeckwith) [inline packages/monitoring-system/data-processor/package.json:32]: Same question as above - why are we keeping this here, but migrating elsewhere?

**Review by JustinBeckwith — APPROVED** [review]

File: packages/failurechecker/package.json (hunk 1 above)
Comment (gr2m) [reply packages/failurechecker/package.json:34]: I think `@octokit/types` can be removed altogether, it doesn't seem to be used at all

File: packages/monitoring-system/data-processor/package.json (hunk 2 above)
Comment (gr2m) [reply packages/monitoring-system/data-processor/package.json:32]: You use 

```ts
import {OctokitResponse} from '@octokit/types';
```

which is specific to the Octokit libraries and goes beyond the scope of GitHub's OpenAPI spec, so you need `@octokit/types` in this case

**Pull request merged**